portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -34,19 +34,28 @@
     
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-     //retrieves comment data from datastore
+     // Get the user input on how many comments to display
+     
+    Integer maxComments = getMaxComments(request);
```

Comment:
What happens if the datastore contains fewer comments than |maxComments| or is empty?